README.md

```diff
@@ -653,6 +653,15 @@ _Default: customscript,runcommand_
 
 The list of extensions which will be excluded from cgroups limits. This should be comma separated. 
 
+#### __Protocol.EndpointDiscovery__
+
+_Type: String_  
+_Default: dhcp_
+
+Determines how the agent will discover the protocol endpoint. Agent will use DHCP by default to discover 
+the wire server endpoint, but if this setting is 'hardcoded' the agent will use the known wireserver ip. 
```

Comment:
```suggestion
the wire server endpoint, but if this setting is 'static' the agent will use the known WireServer address (168.63.129.16). 
```
Or 'known' instead of 'static'